### DISCUSSION TITLE
please help ,why  [FTL] Could not run nuclei: no templates provided for scan  ?

### DISCUSSION THREAD

**Discussion by jeffyds** [projectdiscovery/nuclei-templates#9296] [Q&A] [answered]
[root@ip-10-203-200-173 fuzzing]# nuclei -t xff-403-bypass.yaml -u www.baidu.com

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v3.1.10

                projectdiscovery.io

[INF] Current nuclei version: v3.1.10 (latest)
[INF] Current nuclei-templates version: v9.7.7 (latest)
[WRN] Scan results upload to cloud is disabled.
[INF] Targets loaded for current scan: 1
[INF] No results found. Better luck next time!
[FTL] Could not run nuclei: no templates provided for scan
[root@ip-10-203-200-173 fuzzing]# pwd
/root/nuclei-templates/http/fuzzing

**Comment by ritikchaddha** [comment, marked as answer]
Hello @jeffyds, the template `xff-403-bypass` has a bruteforce tag, and the templates having these tags do not run by default as they are added in nuclei-ignore https://github.com/projectdiscovery/nuclei-templates/blob/main/.nuclei-ignore 
To run these templates, you must use the flag `-itags <tags> `.

Use this command:
`nuclei -id xff-403-bypass -u www.example.com -itags bruteforce `
